spec/models/book_spec.rb

```diff
@@ -0,0 +1,115 @@
+require "rails_helper"
+
+describe Book do
+  before do
+    Book.delete_all
+  end
+
+  describe ".ordered" do
+    it "orders records by name ASC" do
+      beloved = Book.create!(name: "Beloved", isbn: 1987654321)
+      animal_farm = Book.create!(name: "Animal Farm", isbn: 1234567890)
```

Comment:
Sometime later, we'll use [factory_bot](https://github.com/thoughtbot/factory_bot) to make building objects easier. For now, it's sufficient to build them manually as that gives you time to familiarise with lower-level active record methods.